### PR TITLE
Merge release 2.8 to main

### PR DESCRIPTION
**Pull request by lamida**
In this PR I'm merging the release-2.8 branch to main branch after 2.8.0-rc.2. Please merge this PR using a merge commit or by using tools/release/merge-approved-pr-branch-to-main.sh script.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,12 @@
 
 * [CHANGE] Ruler: changed ruler autoscaling policy, extended scale down period from 60s to 600s. #4786
 
+## 2.8.0-rc.2
+
+### Grafana Mimir
+
+* [ENHANCEMENT] Ruler: Improve rule upload performance when not enforcing per-tenant rule group limits. #4828
```
Comment (krajorama) [inline CHANGELOG.md:32]: This line is ok, but should be remove from line 12 above

**Review by pstibrany — APPROVED** [review]
Happy to merge after resolving Krajo's comment.

**Comment by lamida** [comment]
@pstibrany @krajorama duplicated changelog is removed.

**Comment by pstibrany** [comment]
> @pstibrany @krajorama duplicated changelog is removed.

Thanks, merged.

**Comment by pstibrany** [comment]
```console
% ./tools/release/merge-approved-pr-branch-to-main.sh merge-release-2.8-to-main
Already on 'main'
Your branch is up to date with 'origin/main'.
HEAD is now at 860923900 Fix a small typo in the internal contributing docs (#4840)
Merge made by the 'ort' strategy.

Merge successful, pushing to origin...

Enumerating objects: 1, done.
Counting objects: 100% (1/1), done.
Writing objects: 100% (1/1), 247 bytes | 247.00 KiB/s, done.
Total 1 (delta 0), reused 0 (delta 0), pack-reused 0
To https://github.com/grafana/mimir
   860923900..610bf84be  HEAD -> main
```

**Pull request merged**